package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "node-jq",
-  "version": "0.0.0-development",
```

Comment:
Don't remove the version because it will produce a warning with every npm command.